admin/src/pages/View/components/NavigationManager/Form/index.tsx

```diff
@@ -24,6 +25,10 @@ export const Form = ({
   isLoading,
   validationSchema,
 }: Props) => {
+  const initialValues: Navigation= {
+    id: get(navigation, "id", ""),
+    name: get(navigation, "name", ""),
+  }
```

Comment:
Why this change happened?

Reply:
Types mismatched. Props define `navigation` as `Partial<Navigation>` and `onChangeBase` as `(n: Navigation)=> void`. This creates conflict inside Formik and prints out error for initialValues.

Reply:
makes sense